  INFO root [2019-09-20 09:26:54,025] - 
  INFO root [2019-09-20 09:26:54,027] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-20 09:26:54,033] - Resmgr::initialize: KBE_ROOT=D:/Work/Svr/KBEngine_Bate/
  INFO root [2019-09-20 09:26:54,034] - Resmgr::initialize: KBE_RES_PATH=D:/Work/Svr/KBEngine_Bate/kbe/res/;D:/Work/Svr/KBEngine_Bate/server_assets/;D:/Work/Svr/KBEngine_Bate/server_assets/scripts/;D:/Work/Svr/KBEngine_Bate/server_assets/res/
  INFO root [2019-09-20 09:26:54,035] - Resmgr::initialize: KBE_BIN_PATH=D:/Work/Svr/KBEngine_Bate/kbe/bin/server/
  INFO root [2019-09-20 09:26:54,054] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.175 ( 192.168.0.175:55572 )
  INFO root [2019-09-20 09:26:54,054] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.175:55572, SOMAXCONN=511.
  INFO root [2019-09-20 09:26:54,054] - server-configs:
  INFO root [2019-09-20 09:26:54,054] - 	internalTcpAddr : 192.168.0.175:55572
  INFO root [2019-09-20 09:26:54,055] - 	componentID : 4001
  WARN root [2019-09-20 09:26:54,395] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20213, errorno=10048
  INFO root [2019-09-20 09:26:54,400] - ---- dbmgr Version: 2.5.7. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 14:21:01 Sep 18 2019. AppID: 4001. UID: 999. PID: 29968 ----
  INFO root [2019-09-20 09:26:54,401] - Server ly_server_programer: Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz with 2097151kB RAM
  INFO root [2019-09-20 09:26:54,401] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-20 09:26:54,401] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-20 09:26:54,577] - Script::install(): is successfully, Python=(3.7.3 (default, Sep 18 2019, 14:20:04) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-20 09:26:54,577] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-20 09:26:54,686] - Dbmgr::initializeEnd: digest(363BB7D9C7C9010E5779213C13D38542)
S_INFO root [2019-09-20 09:26:54,686] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-20 09:26:54,686] - TelnetServer server is running on port 32000
  INFO root [2019-09-20 09:26:54,687] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-20 09:26:54,858] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
54BDDA3DE7C1E7FE490AF7C60150BEEF55AE0DD19C0FBADC8CEF5B4B16C9BEE752C71B407755AFC6D32845341C3DF707D901284C9B21A3D574B7D5B2EA99D0DE702AF0BE039CC3235211A78455C942AE092761EDB38F75FE469C55EE61159A538868C9BB37572DB834850F97203DF92005565C184D9A02F784E6A5CF4E93BF14
 DEBUG root [2019-09-20 09:26:54,861] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 ERROR root [2019-09-20 09:26:56,866] - DBInterfaceMysql::attach: mysql_errno=2003, mysql_error=Can't connect to MySQL server on 'localhost' (10061)
 ERROR root [2019-09-20 09:26:56,867] - DBUtil::createInterface: attach to database failed!
	dbinterface=0x4fee50
	args=interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=no.

 ERROR root [2019-09-20 09:26:56,867] - Dbmgr::initDB(): can't create dbInterface!
  WARN root [2019-09-20 09:26:56,867] - d:\work\svr\kbengine_bate\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 2.18 seconds
 ERROR root [2019-09-20 09:26:56,867] - app::initialize(): initialization failed!
 DEBUG root [2019-09-20 09:26:56,868] - ThreadPool::destroy(): starting size 0.
 DEBUG root [2019-09-20 09:26:57,269] - ThreadPool::destroy(): successfully!
  INFO root [2019-09-20 09:26:57,275] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-20 09:26:57,275] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-20 09:26:57,576] - ThreadPool::destroy(): waiting for thread(2)[0x872e08,0x872f40,], try=1
 DEBUG root [2019-09-20 09:26:57,978] - TPThread::~TPThread(): 0x872e08
 DEBUG root [2019-09-20 09:26:57,978] - TPThread::~TPThread(): 0x872f40
 DEBUG root [2019-09-20 09:26:57,978] - ThreadPool::destroy(): successfully!
 DEBUG root [2019-09-20 09:26:57,978] - Bundle::destroyObjPool(): size 13.
 DEBUG root [2019-09-20 09:26:57,978] - Channel::destroyObjPool(): size 15.
 DEBUG root [2019-09-20 09:26:57,979] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [2019-09-20 09:26:57,979] - UDPPacket::destroyObjPool(): size 15.
 DEBUG root [2019-09-20 09:26:57,979] - EndPoint::destroyObjPool(): size 15.
 DEBUG root [2019-09-20 09:26:57,979] - Address::destroyObjPool(): size 0.
 DEBUG root [2019-09-20 09:26:57,979] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2019-09-20 09:26:57,979] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2019-09-20 09:27:04,782] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [2019-09-20 09:27:57,195] - 
  INFO root [2019-09-20 09:27:57,196] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-20 09:27:57,197] - Resmgr::initialize: KBE_ROOT=D:/Work/Svr/KBEngine_Bate/
  INFO root [2019-09-20 09:27:57,198] - Resmgr::initialize: KBE_RES_PATH=D:/Work/Svr/KBEngine_Bate/kbe/res/;D:/Work/Svr/KBEngine_Bate/server_assets/;D:/Work/Svr/KBEngine_Bate/server_assets/scripts/;D:/Work/Svr/KBEngine_Bate/server_assets/res/
  INFO root [2019-09-20 09:27:57,198] - Resmgr::initialize: KBE_BIN_PATH=D:/Work/Svr/KBEngine_Bate/kbe/bin/server/
  INFO root [2019-09-20 09:27:57,211] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.175 ( 192.168.0.175:55703 )
  INFO root [2019-09-20 09:27:57,211] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.175:55703, SOMAXCONN=511.
  INFO root [2019-09-20 09:27:57,211] - server-configs:
  INFO root [2019-09-20 09:27:57,211] - 	internalTcpAddr : 192.168.0.175:55703
  INFO root [2019-09-20 09:27:57,212] - 	componentID : 4001
  INFO root [2019-09-20 09:27:58,715] - ---- dbmgr Version: 2.5.7. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 14:21:01 Sep 18 2019. AppID: 4001. UID: 999. PID: 16356 ----
  INFO root [2019-09-20 09:27:58,716] - Server ly_server_programer: Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz with 2097151kB RAM
  INFO root [2019-09-20 09:27:58,716] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-20 09:27:58,717] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-20 09:27:58,892] - Script::install(): is successfully, Python=(3.7.3 (default, Sep 18 2019, 14:20:04) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-20 09:27:58,892] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-20 09:27:58,974] - Dbmgr::initializeEnd: digest(363BB7D9C7C9010E5779213C13D38542)
S_INFO root [2019-09-20 09:27:58,974] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-20 09:27:58,974] - TelnetServer server is running on port 32000
  INFO root [2019-09-20 09:27:58,974] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
 ERROR root [2019-09-20 09:27:58,975] - EntityDef::findScriptModule: [Avatar] not found!
 ERROR root [2019-09-20 09:27:58,975] - Dbmgr::initDB(): not found account script[Avatar]!
 ERROR root [2019-09-20 09:27:58,976] - app::initialize(): initialization failed!
  INFO root [2019-09-20 09:27:58,982] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-20 09:27:58,982] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-20 09:27:59,282] - ThreadPool::destroy(): waiting for thread(2)[0x9f0140,0x9f09c8,], try=1
 DEBUG root [2019-09-20 09:27:59,684] - TPThread::~TPThread(): 0x9f0140
 DEBUG root [2019-09-20 09:27:59,684] - TPThread::~TPThread(): 0x9f09c8
 DEBUG root [2019-09-20 09:27:59,684] - ThreadPool::destroy(): successfully!
 DEBUG root [2019-09-20 09:27:59,684] - Bundle::destroyObjPool(): size 4.
 DEBUG root [2019-09-20 09:27:59,684] - Channel::destroyObjPool(): size 15.
 DEBUG root [2019-09-20 09:27:59,684] - TCPPacket::destroyObjPool(): size 1.
 DEBUG root [2019-09-20 09:27:59,685] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [2019-09-20 09:27:59,685] - EndPoint::destroyObjPool(): size 15.
 DEBUG root [2019-09-20 09:27:59,685] - Address::destroyObjPool(): size 0.
 DEBUG root [2019-09-20 09:27:59,685] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2019-09-20 09:27:59,685] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2019-09-20 09:28:06,488] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [2019-09-20 09:28:48,344] - 
  INFO root [2019-09-20 09:28:48,344] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-20 09:28:48,346] - Resmgr::initialize: KBE_ROOT=D:/Work/Svr/KBEngine_Bate/
  INFO root [2019-09-20 09:28:48,346] - Resmgr::initialize: KBE_RES_PATH=D:/Work/Svr/KBEngine_Bate/kbe/res/;D:/Work/Svr/KBEngine_Bate/server_assets/;D:/Work/Svr/KBEngine_Bate/server_assets/scripts/;D:/Work/Svr/KBEngine_Bate/server_assets/res/
  INFO root [2019-09-20 09:28:48,346] - Resmgr::initialize: KBE_BIN_PATH=D:/Work/Svr/KBEngine_Bate/kbe/bin/server/
  INFO root [2019-09-20 09:28:48,364] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.175 ( 192.168.0.175:55873 )
  INFO root [2019-09-20 09:28:48,364] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.175:55873, SOMAXCONN=511.
  INFO root [2019-09-20 09:28:48,364] - server-configs:
  INFO root [2019-09-20 09:28:48,364] - 	internalTcpAddr : 192.168.0.175:55873
  INFO root [2019-09-20 09:28:48,364] - 	componentID : 4001
  WARN root [2019-09-20 09:28:48,705] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20295, errorno=10048
  INFO root [2019-09-20 09:28:48,708] - ---- dbmgr Version: 2.5.7. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 11178A1C367680938C8E416B64844ACD. Config: Debug X86. Built: 14:21:01 Sep 18 2019. AppID: 4001. UID: 999. PID: 30288 ----
  INFO root [2019-09-20 09:28:48,709] - Server ly_server_programer: Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz with 2097151kB RAM
  INFO root [2019-09-20 09:28:48,709] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-20 09:28:48,709] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-20 09:28:48,892] - Script::install(): is successfully, Python=(3.7.3 (default, Sep 18 2019, 14:20:04) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-20 09:28:48,892] - Script::install(): KBExtra is successfully!
  INFO root [2019-09-20 09:28:48,976] - Dbmgr::initializeEnd: digest(363BB7D9C7C9010E5779213C13D38542)
S_INFO root [2019-09-20 09:28:48,976] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-20 09:28:48,977] - TelnetServer server is running on port 32000
  INFO root [2019-09-20 09:28:48,977] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2019-09-20 09:28:49,132] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
5905F1A4423EC8E3218AB6D572A19BA0C43AE75F6BD4D100ABC603D94696C46976FD98121FA2C0FF3828DC282AF10E8AC5389868D7174E4E0BBA6A993C586E8ED603B0615B4104D384B21446F40C27439BC60826AD3BAB222A46BAA4C3FDA2A8C7B77B7DCFD3AC82593B731E8701EAD7C86AC733170BDA494ACD4DE27F4BB324
 DEBUG root [2019-09-20 09:28:49,135] - DBInterfaceMysql::attach: connect: 129.28.175.241:3306 starting...
 DEBUG root [2019-09-20 09:28:49,402] - DBInterfaceMysql::attach: successfully! addr: 129.28.175.241:3306
  INFO root [2019-09-20 09:28:49,403] - DBUtil::createInterface[0x20feed8]: interface=default, dbtype=mysql, ip=129.28.175.241, port=3306, currdatabase=kbe_centos, username=kbe_centos, connected=yes.

  INFO root [2019-09-20 09:28:49,403] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-20 09:28:49,403] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2019-09-20 09:28:49,403] - DBInterfaceMysql::attach: connect: 129.28.175.241:3306 starting...
 DEBUG root [2019-09-20 09:28:49,403] - DBInterfaceMysql::attach: connect: 129.28.175.241:3306 starting...
 DEBUG root [2019-09-20 09:28:49,404] - DBInterfaceMysql::attach: connect: 129.28.175.241:3306 starting...
 DEBUG root [2019-09-20 09:28:49,404] - DBInterfaceMysql::attach: connect: 129.28.175.241:3306 starting...
 DEBUG root [2019-09-20 09:28:49,404] - DBInterfaceMysql::attach: connect: 129.28.175.241:3306 starting...
  INFO root [2019-09-20 09:28:49,540] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-20 09:28:49,540] - DBInterfaceMysql::checkErrors: not found Account table(dbInterface=default), reset kbe_* table...
  WARN root [2019-09-20 09:28:49,624] - DBInterfaceMysql::checkErrors: reset kbe_* table(dbInterface=default) end!
 DEBUG root [2019-09-20 09:28:49,659] - DBInterfaceMysql::attach: successfully! addr: 129.28.175.241:3306
 DEBUG root [2019-09-20 09:28:49,660] - DBThread::onStart(): 0x22c0380!
 DEBUG root [2019-09-20 09:28:49,660] - DBInterfaceMysql::attach: successfully! addr: 129.28.175.241:3306
 DEBUG root [2019-09-20 09:28:49,661] - DBThread::onStart(): 0x22bfe30!
 DEBUG root [2019-09-20 09:28:49,686] - DBInterfaceMysql::attach: successfully! addr: 129.28.175.241:3306
 DEBUG root [2019-09-20 09:28:49,687] - DBThread::onStart(): 0x22c0270!
 DEBUG root [2019-09-20 09:28:49,704] - DBInterfaceMysql::attach: successfully! addr: 129.28.175.241:3306
 DEBUG root [2019-09-20 09:28:49,704] - DBThread::onStart(): 0x22c02f8!
 DEBUG root [2019-09-20 09:28:49,709] - DBInterfaceMysql::attach: successfully! addr: 129.28.175.241:3306
 DEBUG root [2019-09-20 09:28:49,709] - DBThread::onStart(): 0x22bfda8!
  INFO root [2019-09-20 09:28:49,729] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-20 09:28:49,765] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-20 09:28:49,823] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-20 09:28:49,864] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-20 09:28:49,903] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-20 09:28:49,909] - syncToDB(): Account->sm_autoLoad(tinyint not null DEFAULT 0).
  INFO root [2019-09-20 09:28:49,937] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-20 09:28:49,938] - DBTask::process(): took 0.24 seconds
sql:(delete from kbe_email_verification where logtime<1568939329 and type=3)
  INFO root [2019-09-20 09:28:50,019] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-20 09:28:50,118] - DBInterfaceMysql::query: successfully!
  INFO root [2019-09-20 09:28:50,198] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-20 09:28:50,201] - DBTask::process(): took 0.46 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2019-09-20 09:28:50,282] - DBTask::process(): took 0.53 seconds
sql:(show index from tbl_Account)
 DEBUG root [2019-09-20 09:28:50,382] - DBInterfaceMysql::dropEntityTableFromDB: tbl_Avatar.
  INFO root [2019-09-20 09:28:50,449] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-20 09:28:50,449] - DBInterfaceMysql::dropEntityTableFromDB: tbl_Food.
  INFO root [2019-09-20 09:28:50,508] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-20 09:28:50,508] - DBInterfaceMysql::dropEntityTableFromDB: tbl_Halls.
  INFO root [2019-09-20 09:28:50,569] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-20 09:28:50,569] - DBInterfaceMysql::dropEntityTableFromDB: tbl_Room.
  INFO root [2019-09-20 09:28:50,627] - DBInterfaceMysql::query: successfully!
 DEBUG root [2019-09-20 09:28:50,628] - DBInterfaceMysql::dropEntityTableFromDB: tbl_Smash.
  INFO root [2019-09-20 09:28:50,684] - DBInterfaceMysql::query: successfully!
  WARN root [2019-09-20 09:28:50,684] - d:\work\svr\kbengine_bate\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 1.71 seconds
  INFO root [2019-09-20 09:28:50,956] - ---- dbmgr is running ----
 DEBUG root [2019-09-20 09:28:50,956] - Components::process(): Request for the process of identity...
  INFO root [2019-09-20 09:28:57,168] - ServerApp::onRegisterNewApp: uid:999, username:ly_server_programer, componentType:interfaces, componentID:3001, globalorderID=-1, grouporderID=-1, intaddr:192.168.0.175, intport:30099, extaddr:nonsupport, extport:0,  from 127.0.0.1:30099/0/0/0.
  INFO root [2019-09-20 09:28:57,168] - Components::addComponent: reset orderLog, uid=999!
  INFO root [2019-09-20 09:28:57,168] - Components::addComponent[interfaces], uid=999, componentID=3001, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-20 09:28:58,177] - Components::findComponents: find logger(1)...
  INFO root [2019-09-20 09:28:58,236] - Components::findComponents: found logger, addr:192.168.0.175:55876
  INFO root [2019-09-20 09:28:58,236] - Components::addComponent: reset orderLog, uid=999!
  INFO root [2019-09-20 09:28:58,236] - Components::addComponent[logger], uid=999, componentID=2001, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-20 09:28:58,248] - Components::process(): Found all the components!
  WARN root [2019-09-20 09:28:58,248] - Forwarding logs to logger[192.168.0.175:55876/0/0/0]...
 ERROR root [2019-09-20 09:47:13,064] - Components::removeComponentByChannel: logger : 2001, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=1712582444840, lastReceivedTime=1712335360326, inactivityExceptionPeriod=599494245)
   OFF root [2019-09-20 09:47:13,068] - The following logs sent to logger failed:
  WARN root [2019-09-20 09:47:13,068] - ==> [2019-09-20 09:47:13 064] TCPPacketReceiver::processPendingEvents(192.168.0.175:55876): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-20 09:47:13,068] - ==> [2019-09-20 09:47:13 065] Components::removeComponentByChannel: logger : 2001, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=1712582444840, lastReceivedTime=1712335360326, inactivityExceptionPeriod=599494245)
